R/as-data-frame.R

```diff
@@ -73,3 +104,96 @@ as.data.frame.CrunchDataFrame <- function (x, row.names = NULL, optional = FALSE
     names(out) <- names(x)
     return(structure(out, class="data.frame", row.names=c(NA, -nrow(x))))
 }
+
```

Comment:
Seems like there should also be a `[` method for CrunchDataFrame that lets you set `order`, but maybe that's too much scope creep for this PR.